routes/api/contacts-router.js

```diff
@@ -0,0 +1,80 @@
+import express from "express";
+import Joi from "joi";
+
+import contactsService from "../../models/contacts.js";
+import { HttpError } from "../../helpers/index.js";
+
+const router = express.Router();
+
+const contactAddShema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
+  phone: Joi.string().required(),
+});
+
+router.get("/", async (req, res, next) => {
+  try {
```

Comment:
functions in router should be only invoked, not defined. Create controller file for this functions